core/src/components/list-header/list-header.ionic.scss

```diff
@@ -4,10 +4,12 @@
 
 :host {
   --background: transparent;
+  --color: #{globals.$ionic-color-neutral-1200};
```

Comment:
Since this color is close to black, the snapshots are going to look very similar.